setup.cfg

```diff
@@ -0,0 +1,36 @@
+[metadata]
+name = qutip-tensorflow
+url = https://qutip.org
+description = TensorFlow backend for QuTiP
+long_description = file: README.md
+long_description_content_type = text/markdown; variant=GFM
+keywords = quantum, physics, dynamics, gpu, linear-algebra
+license = BSD 3-Clause License
+license_file = LICENSE.txt
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Topic :: Scientific/Engineering
+    Operating System :: MacOS
+    Operating System :: POSIX
+    Operating System :: Unix
+    Operating System :: Microsoft :: Windows
+author = Alexander Pitchford, Nathan Shammah, Shahnawaz Ahmed, Neill Lambert, Eric Giguere, Boxi Li, Jake Lishman, Simon Cross
+author_email = qutip-admin@googlegroups.com
+platforms = Linux, Mac OSX, Unix, Windows
+
+[options]
+package_dir =
+    =src
+packages = find:
+include_package_data = True
+install_requires =
+    qutip>=5.0.0.a0
```

Comment:
Great idea! Let's do that in a follow up PR though after this one lands.

Reply:
I left all tensorflow-specific requirements for you to put in yourself, because I don't know which versions you'd want to target, or how you'd want to handle its installation, etc.  If you know for sure the minimum version of TensorFlow that all your stuff will work with is, we can put it in now, but I expected that there might need to be some design and playing around first before you'd know what that was.